bundles/admin/javascript-hooks/header.rst

```diff
@@ -0,0 +1,110 @@
+Header
+======
+
+The header-hook renders a header (the blue bar with the toolbar in it) into your component.
+Moreover it takes care of the tab-handling.
+
+Most often you'll find yourself defining a component which contains the header-hook and
+methods for manipulating the toolbar or other methods which implement behaviour which is
+the same over all tabs.
+
+The header-hook in such a component could look something like:
+
+.. code-block:: javascript
+
+    header: {
+       tabs: {
+           url: 'url/to/tabsData'
+       },
+       toolbar: {
+           languageChanger: true
+           buttons: {
+               save: {},
+               settings: {
+                   options: {
+                       dropdownItems: {
+                           delete: {}
+                       }
+                   }
+               }
+           }
+       }
+       title: 'My title',
+    }
+
+What this hook does is essentially the following:
+
+* It initializes the tabs with the data returned from the defined url. From then on
+  you'll never have to worry about tabs again. The whole routing to and starting your
+  tabs component is handled automatically if the data returned by ``tabs.url`` has
+  the right format.
+* It initializes the toolbar with the defined buttons. For information on how to configure buttons
+  and even how to create your own buttons have look :doc:`here </bundles/admin/sulu-buttons>`.
```

Comment:
I would just write `:doc:/bundles/admin/sulu-buttons`, so that the right title is displayed, might help people to know which page your link links to.